### DISCUSSION TITLE
error: ‘ordered_json’ in namespace ‘nlohmann’ does not name a type

### DISCUSSION THREAD

**Discussion by SrideviRamesh** [nlohmann/json#2380] [Q&A] [answered]
I want to make use of both json and ordered_json in my implementation 
But when i added ordered_json in my code i see this issue
**' error: ‘ordered_json’ in namespace ‘nlohmann’ does not name a type'**

please do help..Thanks in advance

test.cpp

```cpp
#include <nlohmann/json.hpp>
#include <iostream>

using json = nlohmann::json;
using ordered_json = nlohmann::ordered_json;
int main()
{
    ordered_json stringdata;
    stringdata["name"] = "Jan"
    stringdata["age"] = 20
    std::cout << stringdata.dump(4) << std::endl;
}
```

**Comment by nlohmann** [comment, marked as answer]
I cannot reproduce this with Xcode 12 using the current `develop` branch. After adding semicolons to lines 9 and 10, the program compiles and produces

```json
{
    "name": "Jan",
    "age": 20
}
```

**Reply by SrideviRamesh** [reply]
Thanks a lot for replying (sorry for the missing semicolons) ...But this issue still persist for me.. Is there a way to check if am using right nlohmann json version ? I am just wondering how to fix this issue

**Reply by nlohmann** [reply]
Where did you get the header from?

**Reply by SrideviRamesh** [reply]
I found out that am using older version of JSON.. This issue is not recreated when i used JSON for Modern C++ version 3.9.0
Thanks

**Reply by nlohmann** [reply]
Thanks for checking back!
